browser/src/unocommands.js

```diff
@@ -100,7 +100,7 @@ var unoCommandsArray = {
 	'DeleteAuthor':{global:{menu:_('Delete All Comments by This Author'),},},
 	'DeleteAxis':{global:{menu:_('Delete Axis'),},},
 	'DeleteCell':{spreadsheet:{context:_('De~lete...'),menu:_('Delete C~ells...'),},},
-	'DeleteColumns':{global:{context:_('Delete selected columns'),menu:_('Delete ~Columns'),},text:{context:_('Delete selected columns'),menu:_('~Columns'),},},
+	'DeleteColumns':{global:{context:_('Delete selected columns'),menu:_('Delete ~Columns'),},text:{context:_('Delete selected columns'),menu:_('Delete ~Columns'),},},
```

Comment:
I see duplicated `menu` here. Did you use script to generate this? I think it shouldn't be modfied manually.
If we use wrong text somewhere, then might be usefult to add the type specifier so we will use other variant:
`_UNO('.uno:PageDown', 'text', here some variant option)`
https://github.com/CollaboraOnline/online/blob/e07a403d0aa2017d4c6ccf69c05e56d466e9ead5/browser/src/unocommands.js#L528